driver/Cases.jl

```diff
@@ -1123,7 +1123,11 @@ function surface_ref_state(::LES_driven_SCM, param_set::APS, namelist)
     les_filename = namelist["meta"]["lesfile"]
 
     Pg, Tg, qtg = NC.Dataset(les_filename, "r") do data
-        Pg = data.group["reference"]["p0_full"][1] #Pressure at ground
+        try
+            Pg = data.group["reference"]["p0_full"][1] #Pressure at ground
+        catch
+            Pg = data.group["reference"]["p0"][1] #Pressure at ground
+        end
```

Comment:
```suggestion
        pg_str = haskey(data.group["reference"], "p0_full") ? "p0_full" : "p0"
        Pg = data.group["reference"][pg_str][1] #Pressure at ground
```
Let's use `haskey` here. Also, do we need the conditional?